tools/create-symlinks.mjs

```diff
@@ -1,34 +1,40 @@
 import * as fs from "fs";
+import {promises as fsPromises} from "fs";
```

Comment:
I think we can clean up the import handling to a single line - maybe switch to something like `import {promises as fsp, existsSync} from "fs"`? Or just use `fs.promises` instead of `fsPromises` in your function calls and remove this import line.